components/HomepageSection/Hero.jsx

```diff
@@ -25,7 +25,7 @@ const Hero = () => (
     <div className="max-w-screen-xl lg:px-12 mx-auto text-center">
       <SectionHeading
         other="lg:mb-4"
-        size="md:text-[4rem]"
+        size="max-sm:text-6xl md:text-[4rem]"
```

Comment:
can be just `text-6xl md:text-[4rem]` I think, b/c somewhere in between `sm` and `md` the text is still small 
<img width="728" alt="image" src="https://github.com/user-attachments/assets/87b65a67-472c-443c-8a06-c7b570b4039b">

Reply:
Sorry Tanya, do you know which dimensions you were able to get this at? I've checked through the different sizes but I can't see where the font size changes

Reply:
@atepem between sm and md, 640px-768px. not a big deal actually, but on some mobile devices this can be seen if the screen is rotated